npm/index.js

```diff
@@ -15,25 +15,25 @@ module.exports = {
    * Initializes Komito Analytics extension.
    * @param {!Object=} opt_options Optional tracking options.
    */
-  init: function(opt_options) {
-    var root = ('object' === typeof self && self.self === self && self) ||
-               ('object' === typeof global && global.global === global && global);
-    var doc = root.document;
+  init: (opt_options) => {
```

Comment:
```javascript
init: (options = {}) => {

}
```
In this case we will avoid `if` statement:
```javascript
if (opt_options) {}
```

docs: https://www.smashingmagazine.com/2016/07/how-to-use-arguments-and-parameters-in-ecmascript-6/#default-parameters-in-ecmascript-6

Reply:
Thanks for the great suggestion. Default function parameters are really useful ES6 feature, but not for this particular case.
In current case we do not need any default values for `opt_options` parameter.

1. The V8, SpiderMonkey, and Nitro JavaScript engines will allocate memory for default arguments.
2. If we remove `if (opt_options)` statement, this operation will be performed on the next line `for (let key in opt_options)` with an extra type and iterability validations anyway. But with separate falsy check, even TypeScript will only do the rest, avoiding duplicate checks.